style.css

```diff
@@ -41,6 +41,34 @@
     font-size: 18px;
   }
 
+  #nav-ul {
+    position: fixed;
+    right: -200%;
+    top: 90px;
+    font-size: 3rem;
+    text-decoration: none;
+    background-color: var(--color-main);
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 0.3s;
+    list-style: none;
+    flex-direction: column;
+  }
+
+  #nav-ul.show {
+    right: 0%;
+    transition: all 0.5s ease-in-out;
+  }
+
+  header .nav-bar .buttons-c .h-bttn {
+    text-decoration: none;
+    color: white;
+    margin: 16px 0;
+  }
+
```

Comment:
- [ ] Please make the mobile menu background cover the whole page to be like the Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/163064298-afbdf6c8-c54b-4878-8c84-0b81c330d975.png) | ![image](https://user-images.githubusercontent.com/68796232/163064203-fe60845a-29c3-45d1-a3a4-84c4932e3bf8.png)  |